src/includes/sub-header.njk

```diff
@@ -1,3 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<script type=“text/javascript”>
+  window.onload = {
+    let subheaders = document.getElementsByClassname('subheaderA');
+    for (var i = 0; i < subheaders.length; i++) {
+      subheaders[i].addEventListener('keydown', function(e){
+        if(e.keyCode === 13 || e.keyCode === 32) {
+          subheaders[i].setAttribute("aria-label", "Link selected");
+        }
```

Comment:
@maraah1 I'm not understanding why we're adding a keydown listener on subheaders. (Does keydown actually fire for non-editable elements?) Can you explain?

Reply:
@maraah1  mind taking a look at this?

Reply:
@jgw96 This issue was determined not necessary and it should be closed, the code is not in the database anymore since November